satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -377,6 +377,102 @@ read limitoffset (
     orderby desc bucket_usage.rollup_end_time
 )
 
+//--- tracking serial numbers ---//
+
+model serial_number (
+	key    id
+	unique serial_number
+
+	field id            serial
+	field serial_number blob
+
+	field bucket_id     blob
+	field expires_at    utimestamp
+
+	index (
+		fields expires_at
+	)
+)
+
+model used_serial (
+	key    serial_number_id
+	unique serial_number_id storage_node_id 
+
+	field serial_number_id serial_number.id cascade
+	field storage_node_id  blob
+)
+
+// inserting a new serial number
+create serial_number ()
+
+// finding out information about the serial number
+read scalar (
+	select serial_number
+	where  serial_number.serial_number = ?
+)
+
+// deleting expired serial numbers
+delete serial_number (
+	where serial_number.expires_at <= ?
+)
+
+// for preventing duplicate serial numbers
+create used_serial ()
+
+// --- bucket roll up tables --- //
+
+model bucket_bandwidth_rollup (
```

Comment:
now that we have the `bucket_bandwidth_rollup` and `bucket_storage_rollup`, do we still need `bucket_usage` table?

Reply:
Yeah, we can eventually remove them, but first we need to implement the new one.